blocks/embed/embed.js

```diff
@@ -174,18 +174,24 @@ const loadEmbed = (block, grandChilds, link, existingClassList) => {
 };
 
 export default function decorate(block) {
+  let checkYt;
   const link = block.querySelector('a').href;
   const childDiv = block.querySelector('div');
   const grandChilds = childDiv ? childDiv.querySelectorAll('div') : [];
   const placeholder = block.querySelector('picture');
   const existingClassList = block.classList;
+  if (link) checkYt = link.includes('youtube') || link.includes('youtu.be');
   block.textContent = '';
 
   if (placeholder) {
     const wrapper = document.createElement('div');
     wrapper.className = 'embed-placeholder';
     wrapper.innerHTML = '<div class="embed-placeholder-play"><button type="button" title="Play"></button></div>';
     wrapper.prepend(placeholder);
+
+    // add unclickable behaviour to the EmbedYtVideo. Removing the behaviour in delayed.js
+    if (checkYt) wrapper.classList.add('unclickable');
```

Comment:
why this change only when we have a placeholder for the embed block ?
I think this should be independent of placeholder